template1.css

```diff
@@ -0,0 +1,144 @@
+* {
+  margin: 0px;
+  padding: 0px;
+  font-family: "Times New Roman", Times, serif;
+  -webkit-print-color-adjust: exact !important;
+}
+.container {
+  margin-left: auto;
+  margin-right: auto;
+}
+.main {
+  display: grid;
+  grid-template-columns: 44% 1% 55%;
+  padding-left: 2%;
+  padding-right: 2%;
+}
+
+header {
+  display: grid;
+  grid-template-columns: 70% 30%;
+  text-align: center;
+  background-color: rgb(212, 211, 211);
+  height: auto;
+  margin: 20px 0px;
+  box-shadow: 1px 1px 12px black;
+}
+
+.right {
+  padding-left: 2%;
+}
+.name {
+  padding: 40px;
+  text-align: center;
+}
+
+.personal-details {
+  border-bottom: 1px dashed black;
+}
+p {
+  padding-top: 1px;
+}
+
+.skill {
+  border-bottom: 1px dashed black;
+  padding-right: 2%;
+}
+
+.education-details {
+  padding-top: 10px;
+}
+
+.edu-1 {
+  padding: 10px 0px 0px 0px;
+}
+
+.edu-2 {
+  padding: 10px 0px 0px 0px;
```

Comment:
We can use these if you want to apply only one and the other is 0px
padding-top
padding-right
padding-bottom
padding-left